AuctionHouse/Overview.md

```diff
@@ -0,0 +1,129 @@
+> * Bob places a bid of 3 SOL on asset A. In doing so, the marketplace pulls 3 SOL from Bob's wallet to the `BuyerEscrowAccount` PDA. This amount will stay here up until the sale goes through.
+    
+## Executing a Sale
+Once we have a listing and a at least one bid placed for a given asset, a trade can be completed by calling the `executeSale` instruction. 
+
+The `executeSale` instruction is a permission-less crank: in other words, can be executed by anyone without any fee or reward. On the execution of the `executeSale` instruction, two things happen:
+* The Auction House pulls the bid amount stored in the `BuyerEscrowAccount` and transfers this amount to the lister (minus Auction House fees). 
+* The `programAsSigner` PDA, which the Auction House assigned as the **Delegate**, pulls the asset from the lister's wallet (more specifically, from the Token Account in the lister's wallet), and transfers the asset into the bidder's wallet, thus completing the trade.
+![](https://i.imgur.com/gpAX63m.png)
+
+Now that we know how `executeSale` instruction works, lets discuss the three trade scenarios in which the `executeSale` instruction is executed in different ways:
+
+1. *Lister agreeing to a bid*: This is the case when the `executeSale` instruction is executed independently, after a **Buy Order** and a **Sell Order** exist for a given asset.
+
+> Example:
+> * Alice lists an asset A for 5 SOL, thus the Auction House creates a **Sell Order** for the asset A.
+> * Bob places a bid of 3 SOL for the asset A. The Auction House creates a **Buy Order** for the asset A.
+> * Alice agrees to the offer, thus enabling the marketplace to "execute the sale" using the `executeSale` instruction.
+
+2. *"Buying" at list price*: This is the case when the bidder places a bid for an already listed asset, at the list amount itself. In such cases, the `executeSale` instruction is executed in the same transaction in which the bid is made (the **Buy Order** is made), and thus the bidder "buys" the asset.
+
+> Example:
+> * Alice lists an asset A for 5 SOL, thus the Auction House creates a **Sell Order** for the asset A.
+> * Bob places a bid of 5 SOL for the asset A, creating a **Buy Order** for the asset A.
+> * This enables the marketplace to place a bid on the asset and execute the sale in the same transaction, in practice allowing Bob to "buy" asset A.
+
+3. *"Selling" at bid price*: This is the case when a user interested in an unlisted asset places a bid on it. Once this happens, if the asset owner, now interested in selling the asset, lists the asset for the bid amount, the `executeSale` instruction is executed in the same transaction in which the listing is done (the **Sell Order** is made), and thus the lister "sells" the asset.
```

Comment:
Could you simplify this paragraph? It's a bit too technical and I've had to read it several times to grasp the information.